osu.Game/Graphics/UserInterface/OsuDropDownMenu.cs

```diff
@@ -0,0 +1,54 @@
+//Copyright (c) 2007-2016 ppy Pty Ltd <contact@ppy.sh>.
+//Licensed under the MIT Licence - https://raw.githubusercontent.com/ppy/osu/master/LICENCE
+
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Linq;
+using System.Reflection;
+using osu.Framework.Graphics.Transformations;
+using osu.Framework.Graphics.UserInterface;
+using OpenTK;
+
+namespace osu.Game.Graphics.UserInterface
+{
+    public class OsuDropDownMenu<U> : DropDownMenu<U>
+    {
+        protected override DropDownHeader CreateHeader() => new OsuDropDownHeader();
+
+        protected override IEnumerable<DropDownMenuItem<U>> GetDropDownItems(IEnumerable<U> values)
+        {
+            return values.Select(v =>
+            {
+                var field = typeof(U).GetField(Enum.GetName(typeof(U), v));
+                return new OsuDropDownMenuItem<U>(
+                    field.GetCustomAttribute<DescriptionAttribute>()?.Description ?? field.Name, v);
+            });
+
+        }
+
+        public OsuDropDownMenu()
+        {
+            //TODO: breaks padding; figure why.
+            //ContentContainer.CornerRadius = 4;

```

Comment:
@Tom94 gonna need your help here... not sure what's going on :(

likely boundingbox/inflate related.